task09-reactJs/one-piece-home-react/src/Components/Image/MonkeyImage.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import image from '../../../src/Images/monkeydluffy.svg'
+
+const MonkeydLuffyImage = () => {
+  return (
+    <img className='characters_img' src={image} alt="monkey" />
+  )
+}
+
+export default MonkeydLuffyImage;
```

Comment:
Maybe we can create only one image component used for many images in order to reuse code.